blueprints/serializer-test/qunit-rfc-232-files/tests/unit/__path__/__test__.js

```diff
@@ -0,0 +1,24 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { run } from '@ember/runloop';
+
+module('serializer:<%= dasherizedModuleName %>', '<%= friendlyTestDescription %>', function(hooks) {
```

Comment:
Should only be two arguments here (`friendlyTestDescription` and the `function(hooks) {}`...